lib/LANraragi/Plugin/Metadata/Eze.pm

```diff
@@ -6,6 +6,7 @@ use warnings;
 #Plugins can freely use all Perl packages already installed on the system
 #Try however to restrain yourself to the ones already installed for LRR (see tools/cpanfile) to avoid extra installations by the end-user.
 use Mojo::JSON qw(from_json);
+use Time::Local qw(timelocal_posix timegm_posix);
```

Comment:
```suggestion
use Time::Local qw(timegm_modern);
```
You'll have to edit the [cpanfile](https://github.com/Difegue/LANraragi/blob/dev/tools/cpanfile) if you want to add Time::Local to the build. (I'm ok with it)